### PR TITLE
added random integer module to mod_random.py

### LINKED ISSUE #24407: Please expand salt module random
I would like to be able to set my /etc/crontab minute and hour settings 
at random for each virtual machine:

```
# m h dom mon dow user  command
17 *    * * *   root    cd / && run-parts --report /etc/cron.hourly
25 6    * * *   root    test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.daily )
47 6    * * 7   root    test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.weekly )
52 6    1 * *   root    test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.monthly )
```

should be come:
```
# m h dom mon dow user  command
random_min *    * * *   root    cd / && run-parts --report /etc/cron.hourly
random_min random_hour    * * *   root    test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.daily )
random_min random_hour    * * 7   root    test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.weekly )
random_min random_hour    1 * *   root    test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.monthly )
```

only thing within salt templating engine for files is wempy, mako and jinja,
but none can do an random number from a max number.

but python self can do this,

### PR DESCRIPTION
**Pull request by aboe76**
fixes #24407 this will help solve it

now you can choose random.rand_int 1 24  to get a random number between the numbers.
this can be used inside salt sls and jinja

### CODE REVIEW

File: salt/modules/mod_random.py [hunk 1]
```diff
@@ -122,3 +123,21 @@ def shadow_hash(crypt_salt=None, password=None, algorithm='sha512'):
         salt '*' random.shadow_hash 'My5alT' 'MyP@asswd' md5
     '''
     return salt.utils.pycrypto.gen_hash(crypt_salt, password, algorithm)
+
+def rand_int(start=1, end=10):
+    '''
+    .. versionadded: 2015.5.1
```
Comment (jfindlay) [inline salt/modules/mod_random.py]: This should be 2015.5.3.  Also I'm guessing you want to merge this into the 2015.5 branch? :-)

**Comment by jfindlay** [comment]
@aboe76, there is also a [lint error](https://jenkins.saltstack.com/job/salt-pr-lint-n/6273/violations/file/salt/modules/mod_random.py/).

File: salt/modules/mod_random.py (hunk 1 above)
Comment (aboe76) [inline salt/modules/mod_random.py]: @jfindlay yes you are right, and if the code looks good please merge it in 2015.5
I have change the version

**Comment by jfindlay** [comment]
Sorry, @aboe76, I am confused, do you want this to go into 2015.5.3 or develop?  I am fine with either case.

**Comment by aboe76** [comment]
@jfindlay, i don't want to confuse you, I have submitted it to develop branch, because it's a new feauture, but I would like to see it in the 2015.5.3 release,

**Comment by jfindlay** [comment]
@aboe76, no problem at all.  This is a small change and is only an addition so we can definitely put it into 2015.5.3.  In the future when you have a bugfix to go into a release branch you are welcome to submit it directly to that branch as the branches are regularly merged forward, see https://docs.saltstack.com/en/latest/topics/development/contributing.html#which-salt-branch.

**Comment by rallytime** [comment]
Go Go Jenkins!

**Pull request merged**